src/ui/simulator/windows/analyzer/analyzer.cpp

```diff
@@ -274,7 +192,7 @@ class AnalyzeSourceFolder final : public Yuni::Thread::IThread
     AnalyzerWizard& pForm;
     Mutex pDataMutex;
     String pFolder;
-    AnalyzerWizard::FileMapping* pMapping;
+    AnalyzerWizard::FileMappingPtr pMapping;
 
 }; // class AnalyzeSourceFolder
 
```

Comment:
On line 234, there is 
```c++
auto& mapping = *pMapping;
```
that should be replaced with : 
```c++
auto& mapping = pMapping.get();
```

Reply:
I removed this reference, it was only used twice